app.py

```diff
@@ -5,15 +5,8 @@
 from werkzeug.exceptions import HTTPException
 from error_handling import CustomError, ValidationError, WrongIdError
 
-app = Flask(__name__)
-env = os.environ.get('FLASK_ENV', 'production')
-if env == 'development':
-    app.config.from_object('config.DevConfig')
-else:
-    os.environ['FLASK_ENV'] = 'production'
-    app.config.from_object('config.ProdConfig')
-
-with open(app.config['PATH_FOR_INITIAL_DATA'], "r") as read_file:
+path = os.environ.get('PATH_FOR_INITIAL_DATA', 'restaurants.json')
```

Comment:
`PATH_FOR_INITIAL_DATA` is one of `Config` fields, it should be used here instead of direct access to `os.environ`

Reply:
fixed